jekyll/_cci2_ja/collect-test-data.md

```diff
@@ -77,34 +77,38 @@ steps:
 
 ここで、`path` キーは、JUnit XML テストのメタデータファイルのサブディレクトリが含まれる `working_directory` への絶対パスまたは相対パス、またはすべてのテスト結果が含まれる一つのファイルのパスです。
 
-`path` の値が隠しフォルダでないことを確認してください。 たとえば、`.my_hidden_directory` は無効な形式です。
+`path` の値が隠しフォルダではないことを確認してください。 たとえば、`.my_hidden_directory` は無効な形式です。
 {: class="alert alert-warning"}
 
 ## ストレージ使用量の表示
 {: #viewing-storage-usage }
 
+<<<<<<< HEAD
```

Comment:
This is a git merge conflict that must have gotten inadvertently committed, but we should definitely exclude this commit from being merged until it can be fixed.

Reply:
Ah thanks, fixing now